.editorconfig

```diff
@@ -0,0 +1,80 @@
+root = true
+
+## Repository Configurations
+############################
+[.{git*,editorconfig,*.yml}]
+end_of_line = lf
+indent_style = space
+indent_size = unset
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[.gitmodules]
+indent_style = tab
+
+[.gitkeep]
+insert_final_newline = unset
+
+
+## PureBasic
+############
+[*.{pb,pbi,pbf}]
+charset = utf-8-bom
+indent_style = space
+indent_size = unset
+end_of_line = unset
+insert_final_newline = true
+trim_trailing_whitespace = unset
+
+
+## Markdown GFM
+###############
+[*.md]
+indent_style = space
+indent_size = unset
+end_of_line = unset
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
`trim_trailing_whitespace = true`

This is not good with Markdown because trailing spaces might be important. For example, two spaces at the end of a line force a line break.

Reply:
I'm aware of that but opt to trim them because having trailing spaces noise in files is worst, and using a `<br />` is always an option (e.g. in your README you use that to break lines,)

But if you want I can remove it.

Reply:
Yes, I use `<br>` to have two lines in a table cell. With pure Markdown this is not possible.

Ok, I agree with you that even with Markdown there should be no trailing spaces. I will replace the two spaces at the ends of the lines with `<br>`.